index.html

```diff
@@ -128,6 +128,7 @@ <h2 class="d2l-heading-3">Components</h2>
 			<ul>
 				<li><a href="components/list/demo/list.html">d2l-list</a></li>
 				<li><a href="components/list/demo/list-drag-and-drop.html">d2l-list (drag & drop)</a></li>
+				<li><a href="components/list/demo/list-nested.html">d2l-list (nested)</a></li>
```

Comment:
Adding this secret demo page to the list 😛

Reply:
The drag & drop page also has a nested example. With all the various ways the list can be configured, it's hard to represent each of them alone with a bizzzillion demo pages, but sometimes that what we want to debug an issue. 😄

Reply:
Yeah, would probably be good to consolidate at some point, and then really flesh out those few demos to have all the functionality. In this case I wanted a nested one that already had the header, and I always forget about that nested demo page